src/models/changeLog.ts

```diff
@@ -0,0 +1,25 @@
+import { ChangeLogContract } from "../contracts/apiChangeLog";
+import { Utils } from "../utils";
+
+export class ChangeLog {
+    public readonly id: string;
+    public name: string;
+    public createdDateTime: string;
+    public updatedDateTime: string;
```

Comment:
Models should also have jsdocs. Here's a good cheatsheet: https://devhints.io/jsdoc